src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -531,6 +531,16 @@ protected void markCheckBoxAsUnchecked(WebElement checkBox) {
         }
     }
 
+    /** 'check' the radio button, if it is not already 'checked'.
+     * No action taken if it is already 'checked'.
+     */
```

Comment:
Maybe shift this comment to javadoc style ie.

```
/**
 * 'check' the radio button, if it is not already 'checked'.
 * No action taken if it is already 'checked'.
 */
```

You might have to change the comments for checkboxes accordingly as well though.